### PR TITLE
log waits in puppeteer tests

### PR DESCRIPTION
**Pull request by bkrmendy**
## Description
This PR adds logging to the commenting and collaboration tests, so they are easier to debug if they fail

### CODE REVIEW

File: puppeteer-tests/src/comments/comment-utils.ts [hunk 1]
```diff
@@ -1,47 +1,61 @@
...
 const BRANCH_NAME = process.env.BRANCH_NAME ? `&branch_name=${process.env.BRANCH_NAME}` : ''
 const BASE_URL = process.env.BASE_URL ?? 'http://localhost:8000'
 
+export async function getElementWithSelector(
+  page: Page,
+  selector: string,
+): Promise<ElementHandle<Element> | null> {
+  console.info('waiting for element with selector', selector)
+  const element = await page.waitForSelector(selector)
+  return element!
```
Comment (seanparsons) [inline puppeteer-tests/src/comments/comment-utils.ts:16]: Is this forcing the result to not be null? When the result type allows that.

**Review by seanparsons — APPROVED** [review]

File: puppeteer-tests/src/comments/comment-utils.ts (hunk 1 above)
Comment (bkrmendy) [reply puppeteer-tests/src/comments/comment-utils.ts:16]: good catch, I wanted it force it not to be null, I'll update the return type

Comment (bkrmendy) [reply puppeteer-tests/src/comments/comment-utils.ts:16]: removed the `| null` in https://github.com/concrete-utopia/utopia/pull/4789/commits/0dc014050e525ee8bc39999483b35a2a5e9bbf16

Comment (seanparsons) [reply puppeteer-tests/src/comments/comment-utils.ts:16]: I'd recommend using `forceNotNull` as I believe the `!` just coerces the type not the value itself.

**Pull request merged**